tests/test_commands.py

```diff
@@ -13,6 +13,7 @@
 FIXTURES = 'tests/fixtures'
 
 SCRIPT = 'website.py'
+MISHMASH = 'mishmash.py'
```

Comment:
It originated as `bad.py` and I was about to use it more until I realized we'd need individual files with individual problems. Removed, amended.